src/main/java/racingcar/domain/car/Position.java

```diff
@@ -0,0 +1,57 @@
+package racingcar.domain.car;
+
+import java.util.Objects;
+
+import racingcar.domain.movement.MovementStrategy;
+
+public class Position {
+
+    public static final int DEFAULT_POSITION = 0;
+    public static final int MIN_POSITION = 0;
```

Comment:
상수는 되도록 private을 기본으로 사용해주세요.

```suggestion
    private static final int DEFAULT_POSITION = 0;
    private static final int MIN_POSITION = 0;
```